boards/nxp/frdm_mcxw71/frdm_mcxw71.dts

```diff
@@ -137,3 +137,8 @@
 	pinctrl-names = "default";
 	status = "okay";
 };
+
+&nbu {
```

Comment:
This support should also be document in the board documentation file.

Reply:
These files document interfaces :
- boards\nxp\rd_rw612_bga\doc\index.rst
- boards\nxp\frdm_rw612\doc\index.rst
- boards\nxp\frdm_mcxw71\doc\index.rst

The Narrow Band Unit driver manages the common intialization interruption of the HCI and the HDLC_RCP drivers.
It is why the hci has been replaced by nbu.
The hci is no longer the only driver whose interrupt needs to be initialized, the hdlc_rcp driver too.
→ Hence a common nbu driver for the both drivers.